packages/catalog-search/src/CurrentRefinements.jsx

```diff
@@ -14,13 +15,33 @@ import {
   NUM_CURRENT_REFINEMENTS_TO_DISPLAY,
   STYLE_VARIANTS,
   LEARNING_TYPE_PATHWAY,
+  LEARNING_TYPE_COURSE,
+  LEARNING_TYPE_PROGRAM,
 } from './data/constants';
 import {
   useActiveRefinementsAsFlatArray,
 } from './data/hooks';
 import { SearchContext } from './SearchContext';
 import { removeFromRefinementArray, deleteRefinementAction } from './data/actions';
 
+const messages = defineMessages({
+  [LEARNING_TYPE_COURSE]: {
+    id: 'search.facetFilters.filterTitle.course',
+    defaultMessage: 'Course',
+    description: 'Title for the course filter.',
+  },
+  [LEARNING_TYPE_PROGRAM]: {
+    id: 'search.facetFilters.filterTitle.program',
+    defaultMessage: 'Program',
+    description: 'Title for the program filter.',
+  },
+  [LEARNING_TYPE_PATHWAY]: {
+    id: 'search.facetFilters.filterTitle.pathway',
```

Comment:
Same as mentioned above about id format. And please do it on otherwise her places too.